jupyterlab/main.tf

```
@@ -49,17 +58,18 @@ resource "coder_script" "jupyterlab" {
   script = templatefile("${path.module}/run.sh", {
     LOG_PATH : var.log_path,
     PORT : var.port
+    BASE_URL : var.subdomain ? "http://localhost:${var.port}/@${data.coder_workspace_owner.me.name}/${data.coder_workspace.me.name}/apps/jupyterlab" : ""
```

Is this reversed?  With a subdomain, there should be no path, and with a subdomain there should be a path?

---

Also does this needs to be just the path?

```suggestion
    BASE_URL : var.subdomain ? "" : "/@${data.coder_workspace_owner.me.name}/${data.coder_workspace.me.name}/apps/jupyterlab""
```

Because:

```
$ jupyter-lab --no-browser --ServerApp.base_url=http://localhost:3000/foo/bar
[I 2024-10-15 18:37:55.987 ServerApp] Jupyter Server 2.14.2 is running at:
[I 2024-10-15 18:37:55.987 ServerApp] http://localhost:8888/http://localhost:8888/foo/bar/lab?token=50a96431d3bfedeef0a1f656fa5382b0bc07a9ebf5086a79

 ```